<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BarSceneItemEdit</class>
 <widget class="QWidget" name="BarSceneItemEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BarSceneItemEdit</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Bar Ayarları</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Bar Tipi : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxBarType">
          <item>
           <property name="text">
            <string>Dikey</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Yatay</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Dikey Ters</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Yatay Ters</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Arc</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Arc Ters</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Minimum Değer :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditMinBarVal"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Maximum Değer :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditMaxBarVal"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Bar Kalınlığı : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSliderBarKalinligi">
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>512</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditKalinlik"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Bar uzunluğu:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSliderBarLength">
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>1024</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditBarLength"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonSelectColor">
        <property name="text">
         <string>Barın Rengini Seç</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Arc Radius :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSliderArcRadius">
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>400</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditArcRadius"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Bar Radius :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSliderBarRadius">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditBarRadius"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_14">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Arc Bend :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSliderArcBend">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditArcBend"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Arc Start Angle :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSliderArcStartAngle">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>360</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditArcStartAngle"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Bar Genel Css 
 (Bar rengi hariç)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEditCss"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Min</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSliderMinMaxDeneme">
          <property name="pageStep">
           <number>1</number>
          </property>
          <property name="sliderPosition">
           <number>99</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Max</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelPercentage">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
